app/templating/summary/question.py

```diff
@@ -98,7 +114,7 @@ def _build_checkbox_answers(answer, answer_schema):
                 else:
                     multiple_answers.append(CheckboxSummaryAnswer(label=option['label'],
                                                                   should_display_other=False))
-        return multiple_answers
+        return multiple_answers if multiple_answers != [] else None
```

Comment:
I'd be tempted to use `if not multiple_answers` as per https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty/53752#53752

But it's a stylistic opinion so I'm not fussed

Reply:
Agreed, but I think this can just be `return multiple_answers or None`? Since empty list is Falsey

Reply:
This is what I was meant to do 👀